data/json/npcs/isolated_road/isolated_road_jay_convert.json

```
@@ -0,0 +1,1315 @@
+          { "u_add_var": "u_waiting_on_rounds", "type": "timer", "context": "artisans", "time": true }
+        ],
+        "topic": "TALK_GUNSMITH_CONVERT_CONFIRM"
+      },
+      {
+        "text": "7.62x51",
+        "condition": {
+          "not": { "u_compare_var": "gunsmith_ammo_from", "type": "number", "context": "artisans", "op": "==", "value": 76251 }
+        },
+        "effect": [
+          { "u_add_var": "gunsmith_ammo_to", "type": "number", "context": "artisans", "value": "76251" },
+          { "u_add_var": "gunsmith_has_rounds_ordered", "type": "dialogue", "context": "artisans", "value": "yes" },
+          { "u_add_var": "u_waiting_on_rounds", "type": "timer", "context": "artisans", "time": true }
+        ],
+        "topic": "TALK_GUNSMITH_CONVERT_CONFIRM"
+      },
+      {
+        "text": "7.62x39",
+        "condition": {
+          "not": { "u_compare_var": "gunsmith_ammo_from", "type": "number", "context": "artisans", "op": "==", "value": 76239 }
+        },
+        "effect": [
+          { "u_add_var": "gunsmith_ammo_to", "type": "number", "context": "artisans", "value": "76239" },
+          { "u_add_var": "gunsmith_has_rounds_ordered", "type": "dialogue", "context": "artisans", "value": "yes" },
+          { "u_add_var": "u_waiting_on_rounds", "type": "timer", "context": "artisans", "time": true }
+        ],
+        "topic": "TALK_GUNSMITH_CONVERT_CONFIRM"
+      },
+      {
+        "text": "5.54x39",
+        "condition": { "not": { "u_compare_var": "gunsmith_ammo_from", "type": "number", "context": "artisans", "op": "==", "value": 545 } },
+        "effect": [
+          { "u_add_var": "gunsmith_ammo_to", "type": "number", "context": "artisans", "value": "545" },
+          { "u_add_var": "gunsmith_has_rounds_ordered", "type": "dialogue", "context": "artisans", "value": "yes" },
+          { "u_add_var": "u_waiting_on_rounds", "type": "timer", "context": "artisans", "time": true }
+        ],
+        "topic": "TALK_GUNSMITH_CONVERT_CONFIRM"
+      },
+      {
+        "text": ".300 AAC",
+        "condition": { "not": { "u_compare_var": "gunsmith_ammo_from", "type": "number", "context": "artisans", "op": "==", "value": 300 } },
+        "effect": [
+          { "u_add_var": "gunsmith_ammo_to", "type": "number", "context": "artisans", "value": "300" },
+          { "u_add_var": "gunsmith_has_rounds_ordered", "type": "dialogue", "context": "artisans", "value": "yes" },
+          { "u_add_var": "u_waiting_on_rounds", "type": "timer", "context": "artisans", "time": true }
+        ],
+        "topic": "TALK_GUNSMITH_CONVERT_CONFIRM"
+      }
+    ]
+  },
+  {
+    "id": "TALK_GUNSMITH_CONVERT_CONFIRM",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "u_compare_var": "gunsmith_ammo_ammount",
+      "type": "number",
+      "context": "artisans",
+      "op": "==",
+      "value": 200,
+      "yes": "Alright 200 rounds, converted minus a 10% service fee, see you in a day.",
```

```suggestion
      "yes": "Alright, that's 200 rounds minus the 10% service fee, they'll be converted in a day.",
```

Phrasing was a bit awkward but I am nitpicking anyway.